### PR TITLE
In-app image uploads

### PR DESCRIPTION
**Pull request by AlexanderOtavka**
# Changes

- Create profile and edit profile pages now allow uploading avatar images without going to AWS

# Testing

1. Start the server
2. Create a new profile, upload an avatar image for that profile
3. Edit a profile, upload a different image for that profile
4. Sign in to the AWS account and delete the images from the bucket, we don't want to fill it with junk

### CODE REVIEW

File: app/views/profiles/edit.html.erb [hunk 1]
```diff
@@ -1,74 +1,77 @@
-<html>
-  <head>  
-    <link href="https://fonts.googleapis.com/css?family=Libre+Franklin" rel="stylesheet">
-  </head>
```
Comment (iangreenleaf) [inline app/views/profiles/edit.html.erb]: Good catch removing this stuff. Are there other views that have the same issue?

Comment (grinnelldennis) [reply app/views/profiles/edit.html.erb]: As far as including fonts, yeah. We're addressing as part of #210

**Comment by grinnelldennis** [comment]
image
The uploader doesn't seem to work for me. I have tried several images both .png and .jpg and all are viewable (thus supposedly non-corrupted) files on my laptop. 

Note: the image upload field is empty in the screenshot because that field resets after form submit. 

Do you know what this is about?

**Comment by grinnelldennis** [comment]
image
Now I get this error when I upload. Any ideas? Also I am wondering how the application's logic will handle multiple profiles with the same name.

**Comment by lynyedongze** [comment]
This seems to work just fine on my workspace. I merged this branch to "staff" and made some minor chnages. The avatar-uploader works for both staff and profiles there. Should we just merge staff branch after the pull request for it is created?

**Comment by grinnelldennis** [comment]
Problem noted from two days ago resolved by swapping AWS access key and access ID

**Pull request merged**